### PR TITLE
Nazanin/website_changes

### PR DESCRIPTION
**Pull request by nazaninreihani**

### CODE REVIEW

File: src/templates/static/affiliates/faq.jsx [hunk 1]
```diff
@@ -41,20 +41,20 @@ const FAQ = () => (
                     <h3>{it.L('What is the definition of a referred client?')}</h3>
                     <p>{it.L('A referred client is someone who has been referred through your unique affiliate link and who has deposited money into their [_1] account. They must fulfil the following criteria:', it.website_name)}</p>
                     <ul className='bullet'>
-                        <li>{it.L('Have not previously been a [_1] customer', it.website_name)}</li>
-                        <li>{it.L('Aged 18 years old and above')}</li>
+                        <li>{it.L('Becoming a [_1] customer for the first time', it.website_name)}</li>
+                        <li>{it.L('Aged 18 years old and above (21 years old and above for clients in Estonia).')}</li>
```
Comment (nazaninreihani) [inline src/templates/static/affiliates/faq.jsx]: Do We need . at the end of this bullet point?

File: src/templates/static/affiliates/faq.jsx [hunk 2]
```diff
@@ -24,15 +24,15 @@ const FAQ = () => (
...
-                    <p>{it.L('Novice traders can use our intuitive platform to learn about trading, practise trading, and gain trading experiences. Seasoned traders can use the [_1] platform and take full advantage of the wide range of trading and analysis tools we have to offer.', it.website_name)}</p>
+                    <p>{it.L('If you’re a broker, we’d like to work with you. You will be introducing your clients to a unique and innovative product: the [_1] trading platform. Your clients will love the [_1] trading platform because we offer a complete binary options trading experience tailored to the needs of an exceptionally wide range of traders.', it.website_name)}</p>
+                    <p>{it.L('New traders can use our intuitive platform to learn about trading, practise trading, and gain trading experience. However, since new traders in the EU are considered as retail clients, they cannot trade binary options due to the current ban in place. Seasoned traders can use the [_1] platform and take full advantage of the wide range of trading and analysis tools we have to offer.', it.website_name)}</p>
```
Comment (jy-binary) [inline src/templates/static/affiliates/faq.jsx]: unnecessary to mention the ban here

File: src/templates/static/affiliates/faq.jsx (hunk 1 above)
Comment (jy-binary) [inline src/templates/static/affiliates/faq.jsx]: did compliance check this 21 thing for Estonia is still required?

File: src/templates/static/affiliates/faq.jsx [hunk 3]
```diff
@@ -74,7 +74,7 @@ const FAQ = () => (
                     </ul>
 
                     <h3>{it.L('Can I offer my referrals an incentive to sign up through my unique affiliate link rather than sign up directly?')}</h3>
-                    <p>{it.L('We prohibit the use of unauthorised incentives, gifts, and payments to encourage customer signups. If we have reason to believe that you have engaged in such activity, we reserve the right to withhold any payments due to you. However, if you have a specific incentive in mind, then please contact your account manager for further discussion and approval.')}</p>
+                    <p>{it.L('Offering any kind of incentives to referrals residing in the European Union is strictly forbidden. Furthermore, we prohibit the use of unauthorised incentives, gifts, and payments to encourage customer signups. If we have reason to believe that you have engaged in such activities, we reserve the right to withhold any payments due to you. However, if you have a specific incentive in mind and your referrals do not reside in the EU, then please contact your account manager for further discussion and approval.')}</p>
```
Comment (jy-binary) [inline src/templates/static/affiliates/faq.jsx]: don't use aggressive wording like "strictly forbidden"

File: src/templates/static/why_us.jsx [hunk 4]
```diff
@@ -87,10 +87,10 @@ const WhyUs = () => (
                         </div>
                         <div className='gr-4 gr-12-m'>
                             <ul className='checked'>
-                                <li>{it.L('Find the right trade for every market condition.')}</li>
+                                <li>{it.L('Make a trade prediction for every possible market condition.')}</li>
                                 <li>{it.L('Execute your trade in seconds.')}</li>
                                 <li>{it.L('Monitor your portfolio in real time.')}</li>
-                                <li>{it.L('Sell back trades at market prices.')}</li>
+                                <li>{it.L('Sell back trades at market prices. (Restrictions may apply)')}</li>
```
Comment (jy-binary) [inline src/templates/static/why_us.jsx]: typography is wrong - you can't start a parenthesis after a full stop, and no need for a capital after the paranthesis

**Comment by kellybinary** [comment]
@nazaninreihani can you update this PR?

**Pull request closed without merging**